src/Rules/MazeWithMiceRule.php

```diff
@@ -16,4 +24,14 @@ public function apply(Field $_field): bool
 
         return $_field->value() ? $survive[$numberOfLivingNeighbors] : $born[$numberOfLivingNeighbors];
     }
+
+    /**
+     * Returns the description of the Rule.
+     * This is used to list all rules if the argument ruleList is set.
+     * @return string description.
+     */
+    public function description(): string
+    {
+        return "Like maze with mice moving in it.";

```

Comment:
That sounds super funny! You must show me that in action! Send me a video in Rocket.Chat!